dlhub_sdk/models/servables/python.py

```diff
@@ -135,6 +141,13 @@ def create_model(cls, path, method, function_kwargs=None) -> 'PythonClassMethodM
             'class_name': class_name
         })
 
+        if auto_inspect:
```

Comment:
sine this is used twice in this file, it looks like it should be a little helper function?